test/end-to-end.js

```diff
@@ -970,6 +977,58 @@ describe('end-to-end', function() {
       });
     });
 
+    describe('error handling in dev/prod', function() {
+      it('includes sensitive error details in development mode', function(done) {
+        var app = loopback({ localRegistry: true, loadBuiltinModels: true });
+        var bootOptions = {
+          appRootDir: SANDBOX,
+          env: 'development'
+        };
+        boot(app, bootOptions, function(err) {
+          if (err) return done(err);
+          app.listen(0, function() {
```

Comment:
No need to call `app.listen(0)`, `request(app)` takes care of that automatically.